addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -13,16 +13,38 @@
 
     @include print-variable('priority-palette-prefix', $o-palette-priority-prefix);
 
-    // Need the colors and name of each color palette
+    // Need the colors, name, menu background color and footer background color of each color palette
     $palette-names: ();
+    $default-presets: ();
+    $bg-attrs: 'menu', 'footer';
+    @each $attr in $bg-attrs {
+        $default-presets: map-merge($default-presets, ($attr: map-get($o-base-color-palette, $attr)))
+    }
+
     @each $palette-name, $palette in $o-color-palettes {
         @if $palette-name != 'user-palette' {
             @each $key, $value in $palette {
                 @include print-variable('o-palette-#{$palette-name}-#{$key}', $value);
             }
             $palette-names: append($palette-names, $palette-name);
+
+            @each $attr-name, $default-preset-idx in $default-presets {
+                $preset-idx: o-safe-get($palette, $attr-name, $default-preset-idx);
+                $preset: nth($o-color-combinations, $preset-idx);
+                $bg: o-safe-get($palette, 'o-cc#{$preset-idx}-bg', map-get($preset, 'bg'));
+                $bg: o-safe-get($palette, $bg, $bg);
+                @include print-variable('o-palette-#{$palette-name}-#{$attr-name}-bg', $bg);
+            }
         }
     }
+
+    // Need the palette colors used as default menu and footer background color
+    @each $attr-name, $default-preset-idx in $default-presets {
+        $preset: nth($o-color-combinations, $default-preset-idx);
```

Comment:
Also throughout this whole PR, if you could avoid the term "preset" in your variable it is confusing :grimacing: There are "palette" and "color combination / cc" but sometimes we use "preset" instead of palette AND/OR color combination when we speak about it among us... but in the code let's stick to palette or color combination everywhere so we know which one we mean for sure.